cdc/capture/capture.go

```diff
@@ -430,6 +430,9 @@ func (c *Capture) campaignOwner(ctx cdcContext.Context) error {
 				c.MessageRouter.RemovePeer(captureID)
 			})
 		}
+		pc, _ := pdutil.NewPlacementClient(ownerCtx, c.PDClient)
+		go pc.Run(20 * time.Second)
+		defer pc.Close()
```

Comment:
defer function won't be triggered until the for loop exits, is this expected?

Reply:
This is a bug.